### DISCUSSION TITLE
When boards will be available for sale?

### DISCUSSION THREAD

**Discussion by makerspet** [makerspet/support#2] [Support Q&A] [answered]
When Maker's Pet boards/kits will be available for purchase?

**Comment by makerspet** [comment, marked as answer]
We are setting up mass production right now, May 09 2024. We expect first boards available for sale in USA by end of May 2024.

Selling to worldwide outside of USA will follow ASAP after that once we obtain CE marking. We also need distributors (so you don't have to pay expensive international shipping).

If you need a board urgently, let us know - we can (temporarily) lend you a prototype board at no charge (only the shipping fee).

**Comment by makerspet** [comment]
In the meantime, if you are comfortable with a little hardware wiring, you can use an ESP32 30-pin breakout board like this one. Please search Google, Amazon, eBay or AliExpress for "ESP32 30-pin breakout board".
 ![esp32-30pin-breakout-board](https://github.com/makerspet/support/assets/143911662/6acbb91e-a816-412b-9537-ddde8867f484)

You will also need a resistor kit like this one. Search for "resistor kit through-hole".
![resistor-kit-through-hole](https://github.com/makerspet/support/assets/143911662/2196f362-37fa-4f75-8772-1b7e7e701e9d)

- If you are using 24V brushless motors, you can use [this schematic](https://github.com/makerspet/pcb/blob/main/esp32_breakout/output/esp32_breakout_schematic.pdf) for wiring.
- If you are using 6V, 9V or 12V brushed motors (including N20), you can use [this schematic](https://github.com/makerspet/pcb/blob/main/esp32_combo/output/esp32_combo_schematic.pdf)
- If you are using YDLIDAR X4, X2, X2L, X3 PRO or RPLIDAR A1 or LDROBOT LD14P or CAMSENSE X1 LiDAR, you can connect those directly to the ESP32 breakout board
- If you are using LDS02RR LiDAR, you will need an [extra circuit](https://github.com/makerspet/pcb/blob/main/lds02rr_adapter/output/LDS02RR_adapter_schematic.pdf) wired up for LDS02RR to operate
- Same goes for Neato XV11 and Delta LiDARS, here is the [additional circuit](https://github.com/makerspet/pcb/blob/main/neato_delta_adapter/output/neato_delta_adapter_schematic.pdf) required for these LiDARs to operate

**Comment by kaiaai** [comment]
Hi,
  I've manufactured the boards and they are available now https://makerspet.com/store
